src/assets/icons/reply.svg

```diff
@@ -1,6 +1,7 @@
 <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" width="24" height="24">
-  <g fill="none" stroke="currentColor" stroke-width="4" stroke-linecap="round"  stroke-linejoin="round">
-    <path d="M20,12H11C2,12,2,5,2,5"/>
-    <polyline points="15 5 22 12 15 19"/>
+  <g fill="none" stroke-width="4" stroke-linecap="round"
+     stroke-linejoin="round" stroke-miterlimit="10">
```

Comment:
Does `stroke-miterlimit` actually have a visible effect?

Reply:
Not that I could tell! Removed.